searx/engines/ansa.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""Engine for Ansa, Italy's oldest news agency.
+
+To use this engine add the following entry to your engines
+list in ``settings.yml``:
+
+.. code:: yaml
+
+  - name: ansa
```

Comment:
This is already part of `settings.yml`, so I don't think we need to explain that explicitely here.

The engine can be enabled like any other engine, so people should read the SearXNG documentation about it at https://docs.searxng.org/admin/settings/settings_engines.html instead.

Reply:
Sorry @Bnyro I don't think I properly get what you mean here:
are you suggesting we remove this?
 `.. code:: yaml
  - name: ansa
    engine: ansa
    shortcut: ans
    disabled: false`

Reply:
Yeah, exactly.